### PR TITLE
Add in simplified page to manage submissions

### LINKED ISSUE #141: Make the Admin leaderboards more informative
The admin leaderboards don't show information about the MCW dataset nor the name of the tool.

Suggestion: Mirror the content of the "production" leaderboard to the admin leaderboard.

### PR DESCRIPTION
**Pull request by thomasyu888**
Fixes #141

### CODE REVIEW

**Comment by tschaffter** [comment]
@thomasyu888 One lint issue: `wiki/611762-SubmissionTrackingAdmin.md:1 MD041/first-line-heading/first-line-h1 First line in a file should be a top-level heading [Context: "# Submission Tracking"]`

File: wiki/611762-SubmissionTrackingAdmin.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Submission Tracking
```
Comment (thomasyu888) [inline wiki/611762-SubmissionTrackingAdmin.md:5]:
```suggestion
<!-- markdownlint-disable-next-line first-line-h1 -->
# Submission Tracking
```

**Comment by tschaffter** [comment]
@thomasyu888 It's preferable to use the format adopted in the other pages rather than disabling the lint rule. Here is a random page that you can use as example https://raw.githubusercontent.com/nlpsandbox/nlpsandbox.io/staging/wiki/604836-UsersTeams.md

**Comment by thomasyu888** [comment]
@tschaffter  Done

**Comment by tschaffter** [comment]
@thomasyu888 H1 titles (`# My H1 title`) - and even H2 - are as big as the page titles displayed by Synapse on the top of each page. For this reason the largest title used by existing pages in H3 and the title of the page displayed by Synapse is not repeated in the document. Here you could simply remove the title `# Submission Tracking`. An alternative solution would be for Synapse to not display the title of the page automatically.

**Comment by tschaffter** [comment]
Anyway I'm merging this PR so you can visualize the current results.

**Pull request merged**